source/pkg/adapter/adapter.go

```diff
@@ -59,25 +64,33 @@ func NewAdapter(ctx context.Context, processed adapter.EnvConfigAccessor, ceClie
 }
 
 func (a *Adapter) Start(ctx context.Context) error {
+	quitChannel := make(chan os.Signal, 1)
+	signal.Notify(quitChannel, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
this is not needed: `ctx` is cancelled (ie. ctx->Done() is closed) when receiving SIGINT and SIGTERM signals